doc/build-unix.md

```diff
@@ -24,145 +22,131 @@ make
 make install # optional
 ```
 
-This will build bitcoin-qt as well if the dependencies are met.
+如果相依套件正確的話，這也會建立 bitcoin-qt。
 
 Dependencies
 ---------------------
 
-These dependencies are required:
+需要的相依套件:
 
- Library     | Purpose          | Description
+ 函式庫      | 目的             | 描述
  ------------|------------------|----------------------
- libssl      | Crypto           | Random Number Generation, Elliptic Curve Cryptography
- libboost    | Utility          | Library for threading, data structures, etc
- libevent    | Networking       | OS independent asynchronous networking
+ libssl      | Crypto           | Random Number Generation, 橢圓曲線密碼學
+ libboost    | Utility          | 函式庫包含執行緒、資料結構...等
+ libevent    | Networking       | 作業系統獨立非同步網路連結
 
-Optional dependencies:
+可選擇的相依套件:
 
- Library     | Purpose          | Description
+ 函式庫      | 目的             | 描述
  ------------|------------------|----------------------
- miniupnpc   | UPnP Support     | Firewall-jumping support
- libdb4.8    | Berkeley DB      | Wallet storage (only needed when wallet enabled)
- qt          | GUI              | GUI toolkit (only needed when GUI enabled)
- protobuf    | Payments in GUI  | Data interchange format used for payment protocol (only needed when GUI enabled)
- libqrencode | QR codes in GUI  | Optional for generating QR codes (only needed when GUI enabled)
- univalue    | Utility          | JSON parsing and encoding (bundled version will be used unless --with-system-univalue passed to configure)
- libzmq3     | ZMQ notification | Optional, allows generating ZMQ notifications (requires ZMQ version >= 4.x)
+ miniupnpc   | UPnP Support     | 跳防火牆輔助
+ libdb4.8    | Berkeley DB      | 錢包儲存 (只有錢包啟動時需要)
+ qt          | GUI              | 圖形用戶介面工具包 (只有圖形用戶介面啟動時需要)
```

Comment:
I think the term *GUI* do not need to be translated. Just say *GUI* and everyone know what it is.